src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -948,50 +948,49 @@
 
 (defn existing-row-ids
   "Returns a map from value to id for each value that's already in the
-   named database column.
-   `column-transform` is used to modify the sql for the values"
-  [table column values column-transform java-type sql-type]
+   named database column."
+  [table column values]
   (into {}
         (for [{:keys [value id]}
               (if (sutils/postgres?)
-                (query-to-vec
-                  (format
-                    "WITH values AS (SELECT unnest(?) as vals)
-                     SELECT %s AS value, id FROM %s
-                     INNER JOIN values ON values.vals = %s.%s"
-                    (column-transform column) (name table) (name table) column)
-                  (sutils/array-to-param sql-type java-type values))
+                (query-to-vec (format
+                               "WITH values AS (SELECT unnest(?) as vals)
+                                SELECT %s AS value, id FROM %s
+                                INNER JOIN values ON values.vals = %s"
+                               column table column)
+                              (sutils/array-to-param "text" String values))
                 (apply query-to-vec
                        (format "SELECT %s AS value, id FROM %s WHERE %s %s"
-                               (column-transform column) (name table) column (jdbc/in-clause values))
+                               column table column (jdbc/in-clause values))
```

Comment:
this is definitely a quibble, but why remove (name table)? Seemed like that allowed consistency with all the other jdbc functions.